pool.opam

```diff
@@ -35,7 +35,7 @@ depends: [
   "ppx_yojson_conv" {>= "0.14.0"}
   "ppx_variants_conv" {= "v0.14.2"}
```

Comment:
So we could update this package now as well?

Reply:
thanks, done